client/webserver/site/src/js/markets.js

```diff
@@ -1525,53 +1557,103 @@ export default class MarketsPage extends BasePage {
    */
   clearOrderTableEpochSide (tbody, newEpoch) {
     for (const tr of Array.from(tbody.children)) {
-      if (tr.order.epoch && tr.order.epoch !== newEpoch) tr.remove()
+      tr.removeEpochOrders()
     }
   }
 
   /*
    * orderTableRow creates a new <tr> element to insert into an order table.
+     Takes a bin of orders with the same rate, and displays the total quantity.
    */
-  orderTableRow (order, cssClass) {
+  orderTableRow (orderBin, cssClass) {
     const tr = this.page.rowTemplate.cloneNode(true)
-    tr.qty = order.qty
-    tr.order = order
-    const rate = order.rate
+    const { rate, sell } = orderBin[0]
+    const isEpoch = !!orderBin[0].epoch
+    tr.orderBin = orderBin
     bind(tr, 'click', () => {
       this.reportClick(rate)
     })
     let qtyTD
+    const updateQtyTD = () => {
+      const mainDiv = qtyTD.children[0]
+      const [qtyDiv, ordersDiv] = mainDiv.children
```

Comment:
Try using the `data-tmpl`/`Doc.tmplElement` system used throughout, and assigning these elements as properties of the custom class mentioned in my other comment.